web-ts/src/@types/FeedbackRatingType.ts

```diff
@@ -0,0 +1 @@
+export type FeedbackRatingType = 'up' | 'down'
```

Comment:
Why is this in the @types directory and not in the file where the FeedbackRatingType is used?

I understand the @types directory more to be external types, or types for untyped modules.

Reply:
Moved it to FeedbackToolbarItem. Originally I thought the rating type will be used at more places, so I moved it to `@types`.